data/nginx/sites.template-local

```diff
@@ -58,27 +58,28 @@ server {
             proxy_buffers              4 256k;
             proxy_busy_buffers_size    256k;
 
+            client_max_body_size 30m;
+
             rewrite ~/(.*)$ /$1 break;
     }
 }
 
-#### For <kc.$NGINX_HOSTNAME>
+#### For <$KC_HOME.$DOMAIN_NAME>
```

Comment:
To be consistent, it should be `KC_HOSTNAME`

Reply:
Yup, that would cause a bug!
Good catch!